src/yachtzee/buffs.ts

```diff
@@ -167,6 +173,11 @@ export function yachtzeePotionSetup(yachtzeeTurns: number, simOnly?: boolean): n
             tries >= 5 * Math.ceil(yachtzeeTurns / potion.effectDuration())
           ) {
             break;
+          } else if (
+            potion.potion === $item`pocket wish` &&
+            failedWishes.includes(potion.effect())
+          ) {
+            break;
```

Comment:
`break`. 
This is in a `while` loop which repeatedly tries to re-obtain the current effect (up to 5 times), because in the first loop we compute the value/profitability of the potion with PYEC, and without in the subsequent loops (this means that we only need to loop at most twice, although the additional tries are there just in case we failed to get the potion for some reason).
There is no need to repeatedly try wishes if we have identified that the wish has failed (i.e. we cannot wish for such an effect), so we break out of this `while` loop.